drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -231,18 +243,26 @@ def build_assessments(enrollment_scope:, data_collection_stages:, unique_by_info
       # so the UI won't prompt the user to complete intake assessments on these enrollments.
       if generate_empty_intakes && data_collection_stages.include?(1)
         Hmis::Hud::CustomAssessment.transaction do
```

Comment:
Is this transaction still useful here, since the save call is moved out of this block?

Reply:
nope good catch, I'll remove

Reply:
oh, that made me notice another bug. the `enrollments_missing_intakes` check assumes that all the generated intakes have been saved so far, which they haven't at this point. Looks like we didn't have a test case to catch that! I will fix and add one.

Reply:
alright, fixed in https://github.com/greenriver/hmis-warehouse/pull/4114/commits/c475fbe46512dcb41208daa4cd58b3dcc14b3ac5
I updated the test to check that we don't create double intake assessments